### PR TITLE
Implemented margin_cross_entropy in paddle.nn.functional.loss

### LINKED ISSUE #23398: margin_cross_entropy

### PR DESCRIPTION
**Pull request by retrogtx**
# PR Description
Close #23398

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by lucasalavapena** [comment]
@retrogtx Thanks for your PR :smile: . Could you please commit a test too? Also please fix the lint failure (https://github.com/unifyai/ivy/actions/runs/6143619925/job/16667435878?pr=23400) in this PR :smile:

**Comment by retrogtx** [comment]
> @retrogtx Thanks for your PR 😄 . Could you please commit a test too? Also please fix the lint failure (https://github.com/unifyai/ivy/actions/runs/6143619925/job/16667435878?pr=23400) in this PR 😄

on it, have classes rn will do it in a while!

**Comment by retrogtx** [comment]
> @retrogtx Thanks for your PR 😄 . Could you please commit a test too? Also please fix the lint failure (https://github.com/unifyai/ivy/actions/runs/6143619925/job/16667435878?pr=23400) in this PR 😄

What sort of lint does Ivy use? I don't seem to grasp it

**Comment by lucasalavapena** [comment]
> > @retrogtx Thanks for your PR 😄 . Could you please commit a test too? Also please fix the lint failure (https://github.com/unifyai/ivy/actions/runs/6143619925/job/16667435878?pr=23400) in this PR 😄
> 
> on it, have classes rn will do it in a while!
> 
> edit: test added

Your tests are failing! Check the CI (https://github.com/unifyai/ivy/actions/runs/6156145164/job/16705365810?pr=23400) new introduced tests or combined tests. Your PR is done, right? Otherwise mark it as a draft first.

**Comment by retrogtx** [comment]
> > > @retrogtx Thanks for your PR 😄 . Could you please commit a test too? Also please fix the lint failure (https://github.com/unifyai/ivy/actions/runs/6143619925/job/16667435878?pr=23400) in this PR 😄
> > 
> > 
> > on it, have classes rn will do it in a while!
> > edit: test added
> 
> Your tests are failing! Check the CI (https://github.com/unifyai/ivy/actions/runs/6156145164/job/16705365810?pr=23400) new introduced tests or combined tests. Your PR is done, right? Otherwise please mark it as a draft first.

Only the test part remains, rest is fine (I guess), I'm extrememly new to open source so um sorry for not marking it as a draft earlier

**Comment by retrogtx** [comment]
> > > > > @retrogtx Thanks for your PR 😄 . Could you please commit a test too? Also please fix the lint failure (https://github.com/unifyai/ivy/actions/runs/6143619925/job/16667435878?pr=23400) in this PR 😄
> > > > 
> > > > 
> > > > on it, have classes rn will do it in a while!
> > > > edit: test added
> > > 
> > > 
> > > Your tests are failing! Check the CI (https://github.com/unifyai/ivy/actions/runs/6156145164/job/16705365810?pr=23400) new introduced tests or combined tests. Your PR is done, right? Otherwise please mark it as a draft first.
> > 
> > 
> > Only the test part remains, rest is fine (I guess), I'm extrememly new to open source so um sorry for not marking it as a draft earlier
> 
> No worries! Good luck fixing it!



> > > @retrogtx Thanks for your PR 😄 . Could you please commit a test too? Also please fix the lint failure (https://github.com/unifyai/ivy/actions/runs/6143619925/job/16667435878?pr=23400) in this PR 😄
> > 
> > 
> > on it, have classes rn will do it in a while!
> > edit: test added
> 
> Your tests are failing! Check the CI (https://github.com/unifyai/ivy/actions/runs/6156145164/job/16705365810?pr=23400) new introduced tests or combined tests. Your PR is done, right? Otherwise please mark it as a draft first.

How can I see what is exactly wrong with my code? The github tests failure screen provides nothing useful to know the same..
Technically I'm kinda overwhemled by seeing a huge codebase, so I'm looking for every possible way I can nitpick my way to the "exact" problem. The "run_test" does not reveal much. 

I'd be grateful if you could guide me towards what my next steps should be

https://github.com/unifyai/ivy/actions/runs/6157332858/job/16707845326?pr=23400
image
